Accessibility/VirtualContent/explainer.md

```diff
@@ -78,6 +78,7 @@ function end_load_next_section(response) {
   document.getElementById("main").appendChild(...);
 
   // Signal to the user that we're done loading.
+  document.getElementById("main").setAttribute("aria-busy", "false");
```

Comment:
Maybe my code review nitpick hat is on too tight, but: `aria-busy` is more of a signal to the AT than the user, so this doesn't really go with the comment immediately above it.

Reply:
Split this out from the other code comment